app/models/tenant.rb

```diff
@@ -220,6 +220,10 @@ def self.root_tenant
     in_my_region.roots.first
   end
 
+  def self.root_tenant_name
+    @root_tenant_name ||= root_tenant.name
```

Comment:
@isimluk thanks a lot for looking on it, I tried it but it was causing failures in other specs
but after some playing, I realized that during switching rspec contexts the database is cleared out but not classes variables on the models and it was causing most of CI failures
- so I added it to `clear_caches` method after each test here https://github.com/ManageIQ/manageiq/pull/15191/commits/74d6415098806fbd5d652d9d0208e23b9e6b7e36
- and I also found the problem with validation method, described in the commit 
https://github.com/ManageIQ/manageiq/pull/15191/commits/ae666299a784f01f915f9424221320fbd490bfc7